pyproject.toml

```
@@ -5,8 +5,10 @@ authors = [
     {name = "Andrew Svetlov",email = "andrew.svetlov@gmail.com"}
 ]
 readme = "README.md"
-requires-python = ">=3.13"
+requires-python = ">=3.9"
 dependencies = [
+    'exceptiongroup==1.3.0; python_version<"3.11"',
+    'backports-asyncio-queues==0.1.2; python_version<"3.13"'
```

Core packaging metadata mustn't use pins. Lower bounds are acceptable, and occasional known broken version exclusions.
Also, `backports-asyncio-queues` doesn't have any provenance metadata. So we should be careful. I think I've found potential source for it on GH. It must be verified and the project persuaded to at least start using trusted publishing and have other verifiable links to the source. The patch below also makes specs sparse, which is more readable.
```suggestion
    'exceptiongroup; python_version < "3.11"',
    'backports-asyncio-queues; python_version < "3.13"'
```